build/task-package.js

```diff
@@ -40,24 +42,25 @@ const fileChanges = {
     ],
     zips: [
       [path.join(Const.PACKAGED_DIST_DIR, `win${archIfNotx64}-unpacked`),
-       path.join(Const.PACKAGED_DIST_DIR, `tofino-${PLATFORM}-${ARCH}.zip`)],
+       path.join(Const.PACKAGED_DIST_DIR, `${PACKAGENAME}.zip`)],
     ],
   },
   darwin: {
     copies: [
       [path.join(Const.PACKAGED_DIST_DIR, 'mac',
                  `${EXENAME}-${getAppManifest().version}-mac.zip`),
-       path.join(Const.PACKAGED_DIST_DIR, `tofino-${PLATFORM}-${ARCH}.zip`)],
+       path.join(Const.PACKAGED_DIST_DIR, `${PACKAGENAME}.zip`)],
 
       [path.join(Const.PACKAGED_DIST_DIR, 'mac',
                  `${EXENAME}-${getAppManifest().version}.dmg`),
-       path.join(Const.PACKAGED_DIST_DIR, `tofino-${PLATFORM}-${ARCH}.dmg`)],
+       path.join(Const.PACKAGED_DIST_DIR, `${PACKAGENAME}.dmg`)],
     ],
   },
   linux: {
-    zips: [
-      [path.join(Const.PACKAGED_DIST_DIR, 'linux'),
-       path.join(Const.PACKAGED_DIST_DIR, `tofino-${PLATFORM}-${ARCH}.zip`)],
+    moves: [
+      [path.join(Const.PACKAGED_DIST_DIR,
+                 `${APPNAME}-${getAppManifest().version}.tar.bz2`),
```

Comment:
Not sure about the difference, the other pull request for auto updates adds a test that makes sure the two stay in sync so after that it doesn't matter much so I'll just make these consistent.